app/javascript/widget/store/modules/contacts.js

```diff
@@ -1,15 +1,36 @@
 import ContactsAPI from '../../api/contacts';
 import { refreshActionCableConnector } from '../../helpers/actionCable';
 
+const state = {
+  currentUser: {},
```

Comment:
What happens if the user reloads the page, `setUser` would be called only once if there is no change in the data.

Reply:
Added a case to fetch from API